### PR TITLE
Add Beta moment and tests for #5078

### PR DESCRIPTION
**Pull request by michaeloriordan**
Add Beta moment and tests for https://github.com/pymc-devs/pymc/issues/5078

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -1190,6 +1190,13 @@ def dist(cls, alpha=None, beta=None, mu=None, sigma=None, sd=None, *args, **kwar
 
         return super().dist([alpha, beta], **kwargs)
 
+    def get_moment(rv, size, alpha, beta):
+        alpha, beta = at.broadcast_arrays(alpha, beta)
```
Comment (ricardoV94) [inline pymc/distributions/continuous.py]:
```suggestion
```
You don't need this line. The operation below will automatically broadcast the arrays.

**Review by ricardoV94 — COMMENTED** [review]
Looks good. Thanks @michaeloriordan!

Although it seems the pre-commit is complaining about the end of file: https://github.com/pymc-devs/pymc/wiki/Python-Code-Style

**Comment by michaeloriordan** [comment]
Missed the pre-commit step, should be fixed now. Thanks @ricardoV94!

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Merged @michaeloriordan. Looking forward to your next PR!

**Comment by twiecki** [comment]
Thanks @michaeloriordan, and congrats on your first PR!

**Pull request merged**